Matchismo/Model/Card/Card.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Card.h
+//  Matchismo
+//
+//  Created by Javidan Ibrahimov on 28.05.2024.
+//
+
+@import Foundation;
+
+@interface Card : NSObject
+
+@property (strong, nonatomic) NSString *contents;
+@property (nonatomic, getter=isChosen) BOOL chosen;
+@property (nonatomic, getter=isMatched) BOOL matched;
+
+- (int)match:(NSArray *)otherCards;
```

Comment:
Please use NSInteger and specify array type 🙏 
```suggestion
- (int)match:(NSArray *)otherCards;
```